.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+sudo: false
+node_js:
+  - "0.10"
+  - "0.12"
+  - "4.2"
```

Comment:
We don't need to test on different Node versions, since there is no Node code in plugins.